elements/lisk-dpos/src/delegates_info.ts

```diff
@@ -38,6 +38,8 @@ interface DelegatesInfoConstructor {
 
 const _isGenesisBlock = (header: BlockHeader): boolean => header.height === 1;
 const zeroRandomSeed = Buffer.from('00000000000000000000000000000000', 'hex');
+const maxConsecutiveMissedBlocks = 50;
+const maxLastForgedHeightDiff = 260000;
```

Comment:
Even though this numbers are in the LIP maybe these values could be config?

Reply:
It might be, but I want to minimize what is exposed at the beginning, and decide to expose the feature if there is good usecase to it